legend-engine-protocol-aws/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/AwsS3ConnectionLexerGrammar.g4

```diff
@@ -0,0 +1,23 @@
+lexer grammar AwsS3ConnectionLexerGrammar;
+
+import CoreLexerGrammar;
+
+// -------------------------------------- KEYWORD --------------------------------------
+
+// COMMON
+IMPORT:                                     'import';
+NONE:                                       'None';
+
+STORE:                                      'store';
+
+//**********
+// S3 CONNECTION
+//**********
+
+S3_PARTITION:                               'partition';
+S3_REGION:                                  'region';
+S3_BUCKET:                                  'bucket';
+
+AWS:                                        'AWS';
+AWS_CN:                                     'AWS_CN';
+AWS_US_GOV:                                 'AWS_US_GOV';
```

Comment:
Are these the only AWS partition anyone will need?  If not, should we avoid these as enum, and leave them as text/string?